src/renderer/views/Search/Search.js

```diff
@@ -90,6 +91,14 @@ export default defineComponent({
   },
   methods: {
     checkSearchCache: function (payload) {
+      const searchCharLimit = this.$store.getters.getSearchCharacterLimit
+
+      if (payload.query.length > searchCharLimit) {
+        console.warn('Search character limit is: ', searchCharLimit)
+        showToast(i18n.t('Search character limit', { searchCharacterLimit: searchCharLimit }))
```

Comment:
You dont need to import i18n in this file
```suggestion
        showToast(this.$t('Search character limit', { searchCharacterLimit: searchCharLimit }))
```